sftpgo/terraform/aws/deps.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: plural.sh/v1alpha1
+kind: Dependencies
+metadata:
+  description: sftpgo aws setup
+  version: 0.1.0
+spec:
+  dependencies:
+  - name: aws-bootstrap
+    repo: bootstrap
+    type: terraform
+    version: '>= 0.1.1'
+  providers:
+  - aws
```

Comment:
Rename folder to `kube`. Add all providers here with all bootstrap dependencies and marked them as optional.